demo/src/app/components/+typeahead/demos/phrase-delimiters/phrase-delimiters.html

```diff
@@ -0,0 +1,6 @@
+<pre class="card card-block card-header">Model: {{selected | json}}</pre>
+<input [(ngModel)]="selected"
+       [typeahead]="states"
+       [typeaheadSingleWords]="true"
+       [typeaheadPhraseDelimiters]="'&,'"
```

Comment:
if you bind a simple string, use this kind of binding `typeaheadPhraseDelimiters="&,"`